.github/ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,39 @@
-This template is a guideline, not a strict requirement.
-- **Version**: 0.5
-- **Integration**: IntelliJ
-- **Configuration**:
+This template is mostly a guideline, not a strict requirement.
+
+Strict requirements: 
+- explicitly using the latest version of `scalafmt`
+- using the `scalafmt` CLI: `https://scalameta.org/scalafmt/docs/installation.html#cli`
```

Comment:
We do have `This template is mostly a guideline, not a strict requirement.` and then we have strict requirements :sweat_smile: 

I would reword this to:
```suggestion
Please make sure that the issue is observable using the latest version of `scalafmt` and within the `scalafmt` CLI: `https://scalameta.org/scalafmt/docs/installation.html#cli`
```

Reply:
done, suggestion accepted as-is.

Reply:
> We do have `This template is mostly a guideline, not a strict requirement.` and then we have strict requirements 😅

that was why I added `mostly`. a couple of things are strict, the rest is not.